### ISSUE TITLE
Use different Learning rate optimizer 

### ISSUE THREAD

**Issue by Axel-Bravo** [Axel-Bravo/19_kaggle_Histopathologic-Cancer-Detection#7]
## Cosine Annealing 
torch.optim.lr_scheduler.CosineAnnealingLR(optimizer, T_max, eta_min=0, last_epoch=-1)
Source [here](https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html#CosineAnnealingLR)

Takes ideas from [here](https://www.jeremyjordan.me/nn-learning-rate/)

## Limited-memory BFGS:
torch.optim.LBFGS(params, lr=1, max_iter=20, max_eval=None, tolerance_grad=1e-05, ,...
Source [here](https://pytorch.org/docs/stable/_modules/torch/optim/lbfgs.html#LBFGS)

## Stochastic Weight Averaging:
Source [here](https://pechyonkin.me/stochastic-weight-averaging/)
